about.html

```diff
@@ -0,0 +1,106 @@
+    <link rel="stylesheet" href="about.css">
+    <title>HennaHubuk</title>
+</head>
+<body>
+    <body>
+        <header>
+          <i class="fa-solid fa-bars" aria-hidden="true"></i>
+          <div class="top-header">
+              <ul>
+                  <li><a href="#"><i class="fa-brands fa-facebook-f"></i></a></li>
+                  <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                  <li><a href="#">English</a></li>
+                  <li><a href="#">My Page</a></li>
+                  <li><a href="#">Logout</a></li>
+              </ul>
+            </div>
+            <div class="sub-header">
+                <a href="index.html"><img src="./images/logo.jfif" alt="website_logo"></a>
+          <div class="header-right">
+            <nav>
+              <ul class="links">
+                <li><a class="nav-link active" href="index.html">Home</a></li>
+                <li><a class="nav-link" href="#">Program</a></li>
+                <li><a class="nav-link" href="#">Join</a></li>
+                <li><a class="nav-link" href="#">Sponsor</a></li>
+                <i class="fa-solid fa-xmark"></i>
+              </ul>
+              <Span>Special Seat</Span>
+            </nav>
+            </div>
+        </header>
+        <div class="begin-about">
+                <div class="intro-about">
+                    <p>"Hello HennaHub Globe!"</p>
+                    <h1 class="intro-trans-anim">HENNAHUBUK'S CREATIVE DESIGN APPRENTICE SUMMIT <br>2023</h1>
+                </div>
+                <div>
+                    <p class="about-text">
+                       @ HennaHub, we offer henna tattoo of any type for any occasion with over 20 years experience, glitter tattoo and face painting. We also offer henna designs using body paint, patch test of 24 to 48 hours is mandatory before full henna application incase of allergies and sensitive skin to 100% plant product. The henna types we use are: 
+                       <br> Natural red henna and Natural black jagua henna 
+                    </p>
+                </div>
+                <div class="contact-us">
+                    Please Contact us per Email, for further enquiry about HennaHubuk apprentice summit 2023 <br>
+                    <span>hennahubuk@gmail.com</span>
+                  </div>
+                </div>
+                <div class="mid">
+                    <div class="mid-text">
+                      <h4>HennaHub Uk Mehndi Designs and Creativity Logo</h4>
+                      <hr>
+                      <p>The Hennahub Uk Apprentice Summit was inspired by the passion and designs of the trainees graduating in December 2023</p>
+                    </div>
+                    <div class="mid-image">
+                      <img src="./images/HennaHubMid.jpeg" alt="concert logo" class="img-mid">
+                    </div>
+                  </div>
```

Comment:
- [ ] Same here as pointed earlier :point_up_2: Kindly wrap this with a `section` tag.